app.py

```diff
@@ -118,7 +118,7 @@ def editTask(user_id, task_id):
 			return redirect(url_for('main', todo=todo, user_id=user_id))
```

Comment:
please make the same changes in editTask as well. Just make it `redirect(url_for('main', user_id=user_id))`